cognite/client/_api/data_modeling/instances.py

```diff
@@ -468,7 +475,7 @@ def list(
         self,
         instance_type: Literal["node"] = "node",
         include_typing: bool = False,
-        sources: list[ViewId] | ViewId | None = None,
+        sources: list[ViewIdentifier] | ViewIdentifier | None = None,
```

Comment:
```suggestion
        sources: Sequence[ViewIdentifier] | ViewIdentifier | None = None,
```

Reply:
Fixed in new commit. Looks like "list" is used for the "sort" parameter so not sure what to use when

Reply:
On inputs we should be accept as much as possible, so we use Sequence as it is more generic and thus accepts a wider set of types. On return types we want to be as specific as possible so as not to restrict what the user can do with the returned object, so we use `list` which is further down in the type hierarchy.